### PR TITLE
Fix codegen bug. Add comments to TWCoinTypeConfiguration.h

### PR DESCRIPTION
**Pull request by armatusmiles**
## Description
Fix codegen parser bug.
Add comments to TWCoinTypeConfiguration.h for more clarify code

## Types of changes

 * Documentation
* Bug fix (non-breaking change which fixes an issue) 

## Checklist

- [ ] Prefix PR title with `[WIP]` if necessary.
- [x] Add tests to cover changes as needed.
- [ ] Update documentation as needed.

### CODE REVIEW

**Comment by armatusmiles** [comment]
Okay. Clang formatter broken a codegen parse. Need to figure this out.

**Comment by armatusmiles** [comment]
Catched it! We have problems when symbol '*' have align left to type.

For example: 
This is correct declaration:
```
bool TWCoinTypeValidate(enum TWCoinType coin, TWString *_Nonnull address);
```

This is bad:
```
bool TWCoinTypeValidate(enum TWCoinType coin, TWString* _Nonnull address);
```
Error: TWCoinType.h:79 Invalid primitive type 'TWString'

**Comment by armatusmiles** [comment]
Ok, this is working :smile: 
I should write codegen. code for all cases and cover their tests

**Comment by armatusmiles** [comment]
Ready for review

File: include/TrustWalletCore/TWCoinTypeConfiguration.h [hunk 1]
```diff
@@ -12,23 +12,29 @@
...
 
 TW_EXPORT_STRUCT
-struct TWCoinTypeConfiguration { 
+struct TWCoinTypeConfiguration {
     uint8_t unused; // C doesn't allow zero-sized struct
 };
 
+/// Returns stock symbol of coin
 TW_EXPORT_STATIC_METHOD
-TWString *_Nonnull TWCoinTypeConfigurationGetSymbol(enum TWCoinType type);
```
Comment (BSathvik) [inline include/TrustWalletCore/TWCoinTypeConfiguration.h]: It's great that the parser doesn't fail on this condition, but can you use left-pointer alignment here to be consistent with the rest of the headers?

Comment (armatusmiles) [reply include/TrustWalletCore/TWCoinTypeConfiguration.h]: > It's great that the parser doesn't fail on this condition, but can you make pointer alignment here consistent with the rest of the headers?

Why? I use [.clang-format](https://github.com/TrustWallet/wallet-core/blob/master/.clang-format#L9) from this repository where: 'PointerAlignment: Left'

Comment (hewigovens) [reply include/TrustWalletCore/TWCoinTypeConfiguration.h]: Most of code pointer style is `Right`, I think we should update `.clang-format`, this pr is too much for just adding comments. Could you please open a issue for `codegen`? and just add comment in this pr

**Pull request closed without merging**